io-engine/src/grpc/v1/snapshot.rs

```diff
@@ -375,7 +375,16 @@ impl SnapshotRpc for SnapshotService {
                             })
                         }
                     };
-                    // prepare snap config and flush IO before taking snapshot.
+                    if let Some(_b) = UntypedBdev::lookup_by_uuid_str(&args.snapshot_uuid) {
```

Comment:
```suggestion
                    if UntypedBdev::lookup_by_uuid_str(&args.snapshot_uuid).is_some() {
```

Reply:
> 

Accepted. Will update it. Learning Rust best practice :)
Thank you..